pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -84,14 +84,24 @@ func (moa *MultiOrgAlertmanager) SyncAlertmanagersForOrgs(orgIDs []int64) {
 		orgsFound[orgID] = struct{}{}
 
 		existing, found := moa.alertmanagers[orgID]
+
+		_, isDisabledOrg := moa.settings.UnifiedAlertingDisabledOrgs[orgID]
 		if !found {
-			reg := moa.orgRegistry.GetOrCreateOrgRegistry(orgID)
-			am, err := newAlertmanager(orgID, moa.settings, moa.configStore, metrics.NewMetrics(reg))
-			if err != nil {
-				moa.logger.Error("unable to create Alertmanager for org", "org", orgID, "err", err)
+			if !isDisabledOrg {
+				reg := moa.orgRegistry.GetOrCreateOrgRegistry(orgID)
+				am, err := newAlertmanager(orgID, moa.settings, moa.configStore, metrics.NewMetrics(reg))
+				if err != nil {
+					moa.logger.Error("unable to create Alertmanager for org", "org", orgID, "err", err)
+				}
+				moa.alertmanagers[orgID] = am
+				existing = am
+			} else {
+				moa.logger.Debug("skipping creating Alertmanger for disabled org", "org", orgID)
 			}
-			moa.alertmanagers[orgID] = am
-			existing = am
+		}
+
+		if isDisabledOrg {
```

Comment:
We could put this on Line 88 with something like:

```
if _, isDisabledOrg := moa.settings.UnifiedAlertingDisabledOrgs[orgID]; isDisabledOrg {
    continue
}
```

and then remove the if statement around Lines 91-100?

Reply:
I just wanted to log the message of skipping starting the Alertmanager in that case. If you have a better suggestion that includes the logging I'll happily change it!